apps/examples/src/index.tsx

```diff
@@ -43,7 +44,11 @@ const router = createBrowserRouter([
 	},
 	{
 		path: 'end-to-end',
-		element: <EndToEnd />,
+		lazy: async () => ({ element: <EndToEnd /> }),
+	},
+	{
+		path: 'hello-world',
+		lazy: async () => ({ element: <HelloWorld /> }),
```

Comment:
I've added another 'hidden' example, just for our own use at `/hello-world` in the examples app. It was useful to check that my changes still worked in a completely vanilla setup. Our examples app now has some more layout going on, which meant it was hard to fully debug a 'hello world' scenario.